chat/package-lock.json

```diff
@@ -0,0 +1,5557 @@
+{
```

Comment:
Why is this file here? What does it do?

I mean we already have a `package-lock.json`on our project.
Is this related to React? Is it necessary?

Delete.